components/Map/Map.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import districtJson from '../../geoJSON/district.json';
+import subcountyJson from '../../geoJSON/subcounty.json';
+import * as turf from '@turf/turf';
+import * as distance from 'jaro-winkler';
+
+const style = {
```

Comment:
This should be a prop. If you want to give it default values, set them appropriately.

Reply:
Set as default values and pushed